### ISSUE TITLE
Need to select the function of the installation path，Double-click to close the Tab

### ISSUE THREAD

**Issue by xiaoyaocode163** [MicrosoftEdge/WebView2Feedback#1393]
Double-click to close the label, and the right-click menu in the address bar can copy "Webpage Title+URL". I like these two functions very much.
It would be more convenient if you can add it.
Chrome also supports command line calls and dozens of options. IE has never considered such a feature.
But IE can be called with VBS (similar to activex exe), which is equivalent to a multi-function setting as a command line.
If the new version of edge can support activex exe, just like vb6 controls excel.exe, how convenient will it be?
If it is purely added to EDGE, it will affect the opening speed. It is recommended to add an edgeActivex.exe specifically for developers to use.
Develop two more components: com dll, activex control (ocx), which is perfect.

Microsoft has never solved the growing problem of the C drive.
If VS2019, SQL SERVER, EDGE can be installed in other drive letter paths, how good?
EDGE, chrome, there is no way to customize the installation path, I have a headache, I don't like to use it.

Basically, the two browsers in China can be installed in any directory. After reinstalling the operating system, they can be opened and run. In fact, he does not need to write the registry, but only writes the configuration in the ini file.
Green software, no need to install, easy to use by decompressing. Mysql's odbc driver is installed in almost one second, which is too convenient, but it does not support Chinese paths.
miniblink from chromium,it's support windows xp.
Welcome to use 360 ​​speed browser
https://browser.360.cn/ee/
360 security browser-comprehensive protection of Internet security, the common choice of 400 million users
https://browser.360.cn/se/

**Comment by xiaoyaocode163** [comment]
The EDGE download page has no response when I click it. After the download is finished, it prompts "Thank you, close", the prompt is not humane.
Online installation, the installation path was not asked, and it was installed to the C drive without the user's consent, which is very bad.
Wouldn't it be good to add a custom option, or click one-click installation, to give consumers some options?

Installation completion prompt: Start, do some settings, and get stuck for a long time before coming out to the next step.
Here instead, you can display the page directly, pop up a floating dialog box, whether you want to customize the settings.
This aspect is much worse than Google, it is not humane, and the operation is troublesome. Install 30 steps in one click.
The earlier version of EDGE prompts you to run under the administrator authority every time you open it. It is not safe to close the prompt.
Displaying favorites is very troublesome. By default, there is a prompt box that can be closed. Is this OK?

**Comment by champnic** [comment]
Hey @xiaoyaocode163 - these seems like feedback for the Edge browser rather than the WebView2 control. You can use "Shift + Alt + i" to open the feedback tool in Edge. Thanks!
